mParticle-Singular/MPKitSingular.m

```diff
@@ -8,6 +8,12 @@
 // This is temporary to allow compilation (will be provided by core SDK)
 NSUInteger MPKitInstanceSingularTemp = 119;
 
+@interface MPKitSingular() {
+    NSDictionary *ddlLink;
```

Comment:
Can you also add a property for the deep link completion handler that's passed into the kit from the implementing app?